pkg/encryption/chunk_encryption.go

```diff
@@ -0,0 +1,37 @@
+package encryption
+
+import (
+	"github.com/ethersphere/bee/pkg/swarm"
+	"golang.org/x/crypto/sha3"
+)
+
+// ChunkEncrypter encrypts chunk data.
+type ChunkEncrypter interface {
+	EncryptChunk([]byte) (key Key, encryptedSpan, encryptedData []byte, err error)
+}
+
+type ce struct{}
```

Comment:
It can be called chunkEncryptor, for clarity, even the code is short, but requires more reading, than to just read the name.